.gitignore

```diff
@@ -104,6 +104,7 @@ typings/
 # Nuxt.js build / generate output
 .nuxt
 dist
+dist-scalprum
```

Comment:
What's stored here?

Reply:
Ah nvm, thought this was at the root but realized it's the bundle output in each package